logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamBatchWriterImpl.java

```diff
@@ -284,4 +292,8 @@ private void resetEvent() {
     bufferWriterInstance.reset();
     metadataWriterInstance.reset();
   }
+
+  private int computeBatchLength(final int eventsCount, final int eventsLength) {
```

Comment:
:thought_balloon: Extracted as it was important to me that we keep `canWriteAdditionalEvent` and `tryWrite` as consistent as possible together. I'm happy to get other suggestions though.